Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -820,65 +824,105 @@ open LinearMap (BilinMap)
 
 section AssociatedHom
 
-variable [CommRing R] [AddCommGroup M] [Module R M] [AddCommGroup N] [Module R N]
-variable (S) [CommSemiring S] [Algebra S R]
-variable [Module S N] [IsScalarTower S R N]
-variable [Invertible (2 : R)] {B₁ : BilinMap R M R}
+variable [CommRing R] [AddCommGroup M] [Module R M]
+
+/-- If `2` is invertible in `R`, then it is also invertible in `End R M`. -/
+instance [Invertible (2 : R)] : Invertible (2 : Module.End R M) :=
+  map_ofNat (algebraMap R (Module.End R M)) 2 ▸ Invertible.map (algebraMap R (Module.End R M)) 2
```

Comment:
It's not usually a good idea to construct data with `▸`; can you use `Invertible.copy` or similar instead?

Reply:
OK; seems to work.